map/framework.cpp

```diff
@@ -3547,6 +3549,16 @@ void Framework::VisualizeRoadsInRect(m2::RectD const & rect)
   }, kScale);
 }
 
+void Framework::MarkLocalAdsCustomer(search::Result & result) const
+{
+  if (m_localAdsManager.Contains(result.GetFeatureID()))
+    result.SetLocalAdsCustomer(true);
+  // For testing only, should be removed.
+  static int i;
+  if (!(++i % 2))
```

Comment:
If you need testing, add a compile options, say, -DTEST_SEARCH_ADS. Moreover, it's much better to move this logic to local ads manager.